src/common/Assert.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2018-2020 European Spallation Source, ERIC. See LICENSE file
```

Comment:
Just a little nitpicking. Copyright cannot be from 2018 since you only started working on this in 2020.
So should "(C) 2020 European ....". Then if you later change this in say 2021 you can use the dash.